src/main/java/com/calendarfiles/TaskProcessor.java

```diff
@@ -44,7 +44,7 @@
     private ArrayList<Event> eventList; 
 
     public TaskProcessor(){
-        datastore = DataStoreServiceFactory.getDataStoreService(); 
+        //datastore = DataStoreServiceFactory.getDataStoreService(); 
```

Comment:
I'm not sure if this is currently a comment, but if so would recommend getting rid of this line of code.